### ISSUE TITLE
The new AE assign to selected token from ChatMessage should be available to players

### ISSUE THREAD

**Issue by itteerde** [foundryvtt/dnd5e#2938]
The new 3.0 AE assign to selected tokens from ChatMessage should be available to players in order to assign to Actors they own.

**Comment by Fyorl** [comment]
It should be visible if it's a spell they own, allowing players to apply self-cast spells on themselves. Outside of that, I don't think it's an improvement to have players piecemeal applying effects to themselves, vs. the DM applying the effect to all applicable targets in a single operation.

**Comment by thatlonelybugbear** [comment]
> But the system does not support that **currently**.

That last sentence (bold added by me), gives me hope. Proper target handling/damaging/applying effects, would do wonders in my eyes 🤞

**Comment by krbz999** [comment]
Yes, I suppose that brings us to #2847.

**Comment by itteerde** [comment]
consider https://github.com/etiquettestartshere/effectivetray ? Seems to solve it for us.

**Comment by etiquettestartshere** [comment]
My thinking about this, if it is on the 3.1.0 milestone, is that the simplest solution and the only change needed is to just delete line [2079](https://github.com/foundryvtt/dnd5e/blob/dc27eb36d8a18588797949f73a0b79032ae7976b/module/documents/item.mjs#L2079C1-L2079C66) from item.mjs:
```diff
- if ( !game.user.isGM && !token.actor?.isOwner ) return false;
```
This way, applying effects is done in exactly the same way as applying damage: you can apply damage (and so you can apply effects) to any actor you own and have selected, regardless of the source. 

While I personally like my players to be able to apply effects to actors they *don't* control via sockets, I think that would be a pretty big change for the system, and should probably come alongside the (optional) ability to do so with more things, such as damage/healing (a feature I personally wouldn't use).

**Comment by kaelad02** [comment]
> It's still a lot faster for the DM to do it in one operation than to have it done piecemeal by the players.

It's not about speed, but workload. The DM has a lot to do already. By letting players apply their own effects, this would take some load off the DM.

I'd argue it's even faster in the cases when the DM targets a player with an feature that requires a saving throw to avoid some status (e.g. being stunned by a Medusa). The player already has their token selected, then clicks the Saving Throw button in the chat message. If they fail, then can click the button to apply the active effect. When the DM has to do it, they have to switch which token they have selected first to the PC, then click the button. In that case it's actually faster for the player to do it.

**Comment by krbz999** [comment]
> It's not about speed, but workload. The DM has a lot to do already. By letting players apply their own effects, this would take some load off the DM.

Agree. After 6 months with the current implementation, I think I can say that for me it needs an improvement and that improvement _has to be_ to let the players have more agency over their own characters here. Constantly having to remind the GM, "we can't, you have to do it for us," has not been a good user experience.
